java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/websocket/exception/GenericMqttWebSocketClientException.java

```diff
@@ -0,0 +1,19 @@
+package com.cumulocity.generic.mqtt.client.websocket.exception;
```

Comment:
nitpick: 'exception' packages are a code smell IMO (smells of scoping by horizontal cross-cutting class role/nature instead of vertical business domain) which is less of an issue for small APIs, but disgusting in APIs with many types and internal verticals. Less of an issue as it is a subpackage in the right domain (websocket package).